apps/backend/src/services/accounts.js

```diff
@@ -136,23 +136,14 @@ const checkAvailability = async ({ email, shortname }) => {
   return result
 }
 
-async function hydrateDemoData({ userId }) {
-  const user = await UserModel.findById(userId)
-
-  // demo data is added to populate new user accounts
-  const titles = ['Demoquestion SC', 'Demoquestion MC', 'Demoquestion FT', 'Demoquestion NR']
-  const types = ['SC', 'MC', 'FREE', 'FREE_RANGE']
-  const content = [
-    'Which of the following statements is applicable to _KlickerUZH_?',
-    'Which of the following formulas have the form of a Taylor polynomial of some degree $$n$$: $$T_n f(x;a)$$? (multiple answers are possible)',
-    'Describe a main principle of a social market economy.',
-    'Estimate the length of the **longest** river in the world (answer in kilometres).',
-  ]
-  const options = [
-    {
+const DEMO_QUESTIONS = [
```

Comment:
Adapted this to an array of objects so we can map through without using index based lookups